src/ui/components/TestSuite/views/TestMetadata/TestItemTree.module.css

```diff
@@ -1,18 +1,35 @@
 .Tree {
   display: flex;
   flex-direction: column;
-  gap: 0.75rem;
 }
 .Tree[data-nested] {
-  padding-left: 0.75rem;
 }
 
 .List {
   display: flex;
   flex-direction: column;
   gap: 0.5rem;
-  padding-left: 0.75rem;
 }
 
 .Scope {
 }
+
+/* Hierarchy styles */
+
+.Tree > .Tree {
+  font-size: 16px;
+}
+
+.Tree > .Tree > .Tree {
+  color: var(--body-color);
+  font-size: 13px;
+}
+
+.Tree > .Tree > .Tree > DIV {
+  font-size: var(--font-size-small);
+  color: var(--theme-base-60);
+  text-transform: uppercase;
+  font-weight: bold;
+  line-height: 2rem;
+  margin-top: 0.75rem;
```

Comment:
This is pretty fragile.

Reply:
@jonbell-lot23  Can you file a linear issue (FE) to revisit this

Reply:
btw please tag me on Test Suites CSS PRs :)

Reply:
Here's a new ticket: https://linear.app/replay/issue/FE-1475/testsuites-hierarchy-needs-better-css